c++/src/capnp/compat/websocket-rpc.c++

```diff
@@ -118,10 +118,7 @@ kj::Promise<void> WebSocketMessageStream::end() {
           //
           // * https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/close
           // * https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent
-
```

Comment:
http.c++:3603 disallows reason on 1005, I can either allow it there or remove the reason here. Seems like we were just inconsistent before.